### ISSUE TITLE
Documentation: add example of persistent volume for Postfix queue

### ISSUE THREAD

**Issue by lesinigo** [bokysan/docker-postfix#60]
Following the current documentation the container will have ephemeral storage and could loose any unsent email if it gets stopped, removed and recreated.

Documentation should contain an example on how to keep Postfix queue on an externally provided volume to make it persistent.

**Comment by bokysan** [comment]
You are correct -- although the `Dockerfile` maps the `/var/spool/postfix` the documentation itself does not explicitly state this.

Merge requests are welcome :)

**Comment by lesinigo** [comment]
Yep I'm currently testing out your (really well made) container right now, so far without any volume mapped.
I will test volume mapping and queue persistence and send an MR for the docs as soon as time permits 👍

**Comment by Nuru** [comment]
If you set `persistence.enabled: true` then it will use a Persistent Volume to store the queue
